docs/application/dotnet/guides/applications/uiapplication/component-based-app.md

```diff
@@ -295,16 +296,19 @@ Following are the main attributes:
 - `setup-appid`
 
   This attribute is only for the widget component element.
-  This attribute is app id that used when launch widget component element.
+  This attribute is app ID that is used when the widget component element is launched.
+
 
 - `max-instance`
+
   This attribute is only for the widget component element.
-  This attribute limits the number of widget instances concurrently executable for a Web application. When omitted or its value is 0, unlimited number of widget instances are supported. The expected value is integer.
+  This attribute limits the number of widget instances concurrently executable for a web application. When omitted or its value is 0, unlimited number of widget instances are supported. The expected value is an integer.
+
 
 
 ## Prerequisites
 
-To use the functions and data types of the component based application API (in [mobile](../../api/mobile/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) and [wearable](../../api/wearable/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) applications), include the `Tizen.Applications.ComponentBased.Common` module in your application:
+To use the functions and data types of the component based application API (in [mobile](/application/dotnet/api/mobile/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) and [wearable](/application/dotnet/api/wearable/latest/group__COMPONENT__BASED__APPLICATION__MODULE.html) applications), include the `Tizen.Applications.ComponentBased.Common` module in your application:

```

Comment:
Hello @mijin85cho,

The mentions of **mobile** and **wearable** shouldn't have existed in this .NET content.
Do you want me to remove these links and the parts within the brackets as well?

Reply:
Hello, @omar-srbd 
If possible, how about changing it to the URL and hyperlink for relative .NET API?

Reply:
After searching through the .NET APIs, I found two links that might be appropriate for this sentence.

1. https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.Common.html
2. https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.Common.ComponentBasedApplication.html

Which one should I be linking to?

Reply:
"To use the functions and data types of the component based application API (in mobile and wearable applications), include the **Tizen.Applications.ComponentBased.Common** module in your application:

```
using Tizen.Applications.componentbased.common
using Tizen.Applications.componentbased.default  // if you want to use ELFComponentBasedApplication

```
"

So I think https://docs.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.Common.html would be better.

Also, when you change the link, would you please change the sentence form so it can follow the string form like other .NET pages?
For example, as like "To use the methods and properties of the _Tizen.Applications_ namespace, include it in your application:" in https://docs1.stg.tizen.org/staging/1734/application/dotnet/guides/applications/uiapplication/ui-app/.

Reply:
I am currently working on further site fix for the .NET part.
I will make the necessary change to this file and commit the change with the new PR that I will open soon.

Thank you so much for helping me out.

Reply:
So many thanks @omar-srbd!